libs/domain/picking/src/components/picking-lists/picking-lists.component.ts

```diff
@@ -71,7 +71,16 @@ export class PickingListsComponent extends LitElement {
   }
 
   protected renderEmptyLists(): TemplateResult {
-    return html`<p>${i18n('picking.no-picking-lists-found')}</p>`;
+    return html`
+      <div class="no-items-fallback">
+        <oryx-heading as="h4">
+          ${i18n('picking.no-results-found')}
+        </oryx-heading>
+        <div class="img-wrap">
```

Comment:
We can drop this unnecessary `img-wrap` wrapper for `oryx-image` if just improve a bit the styles:

```
.no-items-fallback {
    display: flex;
    flex-direction: column;
    align-items: center;
    height: 100%;
    margin: 45px;
    gap: 16px;
  }

  oryx-image {
    margin-inline: 40px;
    display: inline-block;
  }
```